### PR TITLE
feat: rename add liquidity mutations to deposit

### LINKED ISSUE #2303: Rename liquidity resolvers to use "deposit" instead of "add."
## Context

We have some inconsistent liquidity resolver names as discussed here: https://github.com/interledger/rafiki/pull/2151#discussion_r1406673266

We use "add" in some places and "deposit" in others and decided to standardize on "deposit".

This will break integrator's where they call the existing endpoints and should be released with #1697.

## Todos
- [x] rename `addAssetLiquidity` -> `depositAssetLiquidity` and related names
- [x] rename `addPeerLiquidity` -> `depositPeerLiquidity` and related names
- [x] update frontend
- [x] update postman
- [x] update docs

### PR DESCRIPTION
**Pull request by BlairCurrey**
## Changes proposed in this pull request

Renamed the following and updated related names, frontend, postman, docs for:
- `addAssetLiquidity` mutation -> `depositAssetLiquidity`
- `addPeerLiquidity` mutation -> `depositPeerLiquidity`
- `addedLiquidity` field on `CreateOrUpdatePeerByUrlInput` mutation -> `depositedPeerLiquidity`

## Context

fixes #2303

## Checklist

- [x] Related issues linked using `fixes #number`
- [x] Tests added/updated
- [x] Documentation added
- [x] Make sure that all checks pass
- [x] Postman collection updated

### CODE REVIEW

**Comment by JoblersTune** [comment]
I found more instances of 'add'

- packages/backend/src/payment-method/ilp/auto-peering/errors.ts:` [AutoPeeringError.LiquidityError]: 'Could not add liquidity to peer'`
- packages/backend/src/payment-method/ilp/auto-peering/service.test.ts: `returns error if could not add liquidity during peer creation` and `returns error if could not add liquidity during peer update`
- packages/backend/src/payment-method/ilp/auto-peering/service.ts: `Could not add liquidity to peer`
- packages/backend/src/payment-method/ilp/peer/service.test.ts: `Add Liquidity` and `Can add liquidity to peer`
- packages/backend/src/payment-method/ilp/peer/service.ts: `error trying to add initial liquidity`

**Review by sabineschaller — COMMENTED** [review]
I could not find any more instances of "add" but one test failed for me locally: 
packages/backend/src/graphql/middleware/index.test.ts 
● GraphQL Middleware › lockGraphQLMutationMiddleware › throws error on concurrent call with same key

**Comment by BlairCurrey** [comment]
> I could not find any more instances of "add" but one test failed for me locally: packages/backend/src/graphql/middleware/index.test.ts ● GraphQL Middleware › lockGraphQLMutationMiddleware › throws error on concurrent call with same key

@sabineschaller I could not reproduce this and it's passing here. Perhaps an intermittent failure, or resolved after `pnpm clean && pnpm install`? I'm not seeing anything related to the changes I've made in that test.

**Comment by mkurapov** [comment]
I'm trying to figure out why the deploy preview https://deploy-preview-2304--brilliant-pasca-3e80ec.netlify.app/apis/backend/mutations/#addpeerliquidity still contains the `addPeerLiquidity` mutation

File: packages/backend/src/graphql/schema.graphql [hunk 1]
```diff
@@ -306,8 +310,8 @@ input CreateOrUpdatePeerByUrlInput {
   name: String
   "Account Servicing Entity will be notified via a webhook event if peer liquidity falls below this value"
   liquidityThreshold: UInt64
-  "Initial amount of liquidity to add for peer"
-  addedLiquidity: UInt64
+  "Initial amount of liquidity to deposit for peer"
+  depositedLiquidity: UInt64
```
[the remark this answers is not in the document]
Comment (mkurapov) [reply packages/backend/src/graphql/schema.graphql:314]: Also, can you update the Auto-peering section (example mutation) in the docs to reflect this change please

**Review by mkurapov — APPROVED** [review]

File: packages/backend/src/graphql/schema.graphql (hunk 1 above)
[the remark this answers is not in the document]
Comment (BlairCurrey) [reply packages/backend/src/graphql/schema.graphql:314]: Do we want to differentiate this from the `CreatePeerInput.initialLiquidity` field? Before your edit here the comment description was identical, although the name was different. Just noticing that we could call this `initialLiquidity` as well but perhaps this doesn't make sense in the context of updating.

[the remark this answers is not in the document]
Comment (BlairCurrey) [reply packages/backend/src/graphql/schema.graphql]: updated https://github.com/interledger/rafiki/pull/2304/commits/d708febc4312a65de7c4d31a4d1e77c6467539e5

[the remark this answers is not in the document]
Comment (mkurapov) [reply packages/backend/src/graphql/schema.graphql]: In some cases they are doing different things, so its ok that they are different. Only other change would be to (maybe?) change `initialLiquidity` to `initialLiquidityToDeposit` but I think it's already self explanatory (+ the comment is there as well)

File: packages/documentation/src/content/docs/concepts/interledger-protocol/peering.md [hunk 2]
```diff
@@ -357,7 +357,7 @@ with the input being:
   "input": {
     "peerUrl: "PEER_URL",
     "assetId": "INSERT_ASSET_ID",
-    "initialLiquidity: <optionally, and initial amount of liquidity to provision>
+    "liquidityToDeposit": <optionally, and initial amount of liquidity to provision>
```
Comment (mkurapov) [inline packages/documentation/src/content/docs/concepts/interledger-protocol/peering.md:360]:
```suggestion
    "liquidityToDeposit": <optional amount of liquidity to deposit>
```

File: packages/documentation/src/content/docs/concepts/interledger-protocol/peering.md [hunk 3]
```diff
@@ -125,7 +125,7 @@ Query Variables (substitute the asset ID from the "create asset" response for `I
       "outgoing": {"endpoint": "ilp.othergreatwallet.com", "authToken": "theirtoken"}
     },
     "assetId": "INSERT_ASSET_ID",
-    "initialLiquidity: <optionally, and intial amount of liquity to provision. Liquidity can also be added via the `AddPeerLiquidity` mutation described below>
+    "initialLiquidity: <optionally, and intial amount of liquity to provision. Liquidity can also be deposited via the `DepositPeerLiquidity` mutation described below>
```
Comment (njlie) [inline packages/documentation/src/content/docs/concepts/interledger-protocol/peering.md:128]: I think you are missing a closing quote around `"initialLiquidity(")`

**Comment by sabineschaller** [comment]
> I could not reproduce this and it's passing here. Perhaps an intermittent failure, or resolved after `pnpm clean && pnpm install`? I'm not seeing anything related to the changes I've made in that test.

@BlairCurrey I tried that (again after upgrading pnpm) and still the same issue. But it works in the CI so it must be a me-problem.

**Pull request merged**